Resources/Public/JavaScript/Backend/FormEditor/CheckboxlinkViewmodel.js

```diff
@@ -0,0 +1,132 @@
+// Resources/Public/JavaScript/Backend/FormEditor/CheckboxlinkViewmodel.js
```

Comment:
upper case 'm'. This has to updated in filename itself, too.

Reply:
Done